content/800-data-platform/05-data-browser.mdx

```diff
@@ -0,0 +1,165 @@
+   <img src="https://res.cloudinary.com/prismaio/image/upload/v1657303813/docs/pdp/data-browser-records-actions_v02_znqx9x.png" width="450px" alt="Data Browser - Show and hide records"/>
+
+**Result**
+
+The model is immediately filtered to show or hide records based on your selection.
+
+The **Showing** menu indicates how many records are shown out of how many available records are in the model.
+
+## Editing data
+
+In the model view, you can edit data directly in the model cells. You can add new records as well as delete existing records.
+
+You must confirm every edit operation (add, edit, delete). You must confirm any new records that you add or edits of existing records that you make with the **Save change** button. 
+
+You can accumulate multiple added records and edited cells which you can finalize with the **Save changes** button. You cannot accumulate deletes of records. They always complete immediately after you confirm them. In addition, they also force-save any unconfirmed edits.
+
+You can also discard accumulated changes with **Discard changes**.
+
+### Add a record
+
+1. In the model view, click **Add record**.
+2. Based on the data allowed in each field, type the data for the record.
+
+   | Field data type | Description                                                                                                                                                                                                                                                         |
+   | :--------------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+   |       `A?`       | Text                                                                                                                                                                                                                                                                |
+   |       `#`        | Integer<br /><br />If such a field has `autoincrement()` pre-filled, do not edit the cell and add a number.                                                                                                                                                        |
+   |       `[]`       | List<br /><br />It typically referes to a list of records that exist in another model in the database.                                                                                                                                                                  |
+   |       `{}`       | Relation with a model defined separately in the database.<br /><br />Typically, you need to select the same value as any of the previous records<br />Click the name of the model to see the list of values which you can then select for the related field. |
```

Comment:
I think there might be more of these. DevProd team should be able to tell you.

Reply:
One for example is `Json`, and embedded documented (for MongoDB) [which shows the same as `Json` right now].

Reply:
This list could also be already listed in the "show data" section at the beginning - it describes what type a field/column is.

Reply:
I've added information about the `Json` field.